guide/07-working-with-bigdata/01-introduction.ipynb

```diff
@@ -180,6 +180,13 @@
    "source": [
```

Comment:
<p>In this guide, we have gained familiarity with big data, GeoAnalytics server, its uses and how tools can help us analyze it. In the next guide, we will learn how data can be made accessible to the GeoAnalytics server.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>In this guide, we have gained familiarity with how to utilize GeoAnalytics server to analyze big data. In the next guide, we will learn how data can be made accessible to the GeoAnalytics server.</p><p> </p><p><strong><em>I want to clarify the first sentence a bit.</em></strong></p>